src/gas/gas-util.ts

```diff
@@ -0,0 +1,107 @@
+import { BN } from 'ethereumjs-util';
+import { query, handleFetch, gweiDecToWEIBN, weiHexToGweiDec } from '../util';
+import {
+  GasFeeEstimates,
+  EthGasPriceEstimate,
+  EstimatedGasFeeTimeBounds,
+  unknownString,
+} from './GasFeeController';
+
+const GAS_FEE_API = 'https://mock-gas-server.herokuapp.com/';
+export const EXTERNAL_GAS_PRICES_API_URL = `https://api.metaswap.codefi.network/gasPrices`;
+
+export async function fetchGasEstimates(): Promise<GasFeeEstimates> {
+  return await handleFetch(GAS_FEE_API);
+}
+
+/**
+ * Hit the legacy MetaSwaps gasPrices estimate api and return the low, medium
+ * high values from that API.
+ */
+export async function fetchLegacyGasPriceEstimates(): Promise<{
+  low: string;
+  medium: string;
+  high: string;
+}> {
```

Comment:
is there a reason to don't use the `LegacyGasPriceEstimate` interface?
```ts
import { LegacyGasPriceEstimate } from './GasFeeController';

//...

export async function fetchLegacyGasPriceEstimates(): Promise<LegacyGasPriceEstimate> {
```

Reply:
Good call, Resolved 53250a9